pyanaconda/modules/payload/payload_interface.py

```diff
@@ -27,9 +28,10 @@
 class PayloadInterface(KickstartModuleInterface):
     """DBus interface for Payload module."""
 
-    def CreateLiveOSHandler(self):
-        """Create Live OS payload handler and publish it on dbus.
+    def CreateLiveOSHandler(self) -> ObjPath:
+        """Create Live OS payload handler and publish it on dbus."""
+        return self.implementation.create_live_os_handler()
 
-        # FIXME: This is a temporary solution which will change in the future commits
-        """
-        self.implementation.create_live_os_handler()
+    def GetActiveHandler(self) -> ObjPath:
```

Comment:
This name is on purpose and it is a preparation for the multiple payloads in the future. This will then return the main payload and we don't have to change the API later.